src/components/PageTitle/index.tsx

```diff
@@ -12,21 +12,23 @@ type Props = {
   smaller?: boolean
   subTitle?: string
   title: string
+  focused?: boolean
 }
 
 const PageTitle: FunctionComponent<Props> = ({
   title,
   subTitle,
   smaller,
   className,
+  focused
 }) => {
   const isFullScreen = useSelector<RootState, boolean | undefined>(
     (state) => state.globals.isFullScreen
   )
   const containerRef = useRef<HTMLSpanElement>(null)
 
   useEffect(() => {
-    containerRef.current && containerRef.current.focus()
+    containerRef.current && focused && containerRef.current.focus()
```

Comment:
Can we pick this value from props to a provider so PageTitle avoid to autofocus on other views too instead of only welcome panel ?

Reply:
I'm not sure it's good behavior to be "all in" or not. Some Onfido dev will have to make the say. I agree with you.

Reply:
Yes, I agree that some onfido dev should evaluate this and eventually add the necessary tests with this change. Hope this PR can highlight the issue at least, thks for making it  !

Reply:
We've actually had a need to have the focus not pulled back to the PageTitle in a/some screens/scenarios. See PR #1560 where it is passed down as a prop.

But we have a bit more control internally within our code to choose which screen gets to pass the prop down. We'll need to have a think about how to best make this available as an integrator controllable option since as your discussions in the original issue raised show that integrators don't always start from the Welcome screen and order of steps also differ and an "all in" approach is not ideal IMO.